addons/sale/models/sale_order.py

```diff
@@ -1220,58 +1220,16 @@ def action_unlock(self):
         self.locked = False
 
     def action_cancel(self):
```

Comment:
Check with reviewer is it good idea to merge action_cancel and _action_cancel methods